multiapps-controller-core/src/main/java/org/cloudfoundry/multiapps/controller/core/cf/CloudControllerClientFactory.java

```diff
@@ -46,51 +47,60 @@ private ImmutableCloudControllerRestClientFactory createClientFactory(Applicatio
                                                         .build();
     }
 
-    @Override
-    protected CloudControllerClient createClient(CloudCredentials credentials) {
+    public CloudControllerClient createClient(OAuth2AccessTokenWithAdditionalInfo token) {
```

Comment:
I think it is possible in CloudCredentials to set target of org and space. How this will be possible now?

Reply:
I removed the methods with `CloudCredentials` parameter, because they were not used actually. But could be easily added again if we decide.